ops/services/okta-global/main.tf

```diff
@@ -65,6 +93,21 @@ resource "okta_group" "prime_users" {
   description = "All Prime team members"
 }
 
+resource "okta_group" "simplereport_prod_admins" {
+  name        = "SR-PROD-ADMINS"
+  description = "PRODUCTION Application Administrators for SimpleReport"
+}
+
+resource "okta_group" "simplereport_stg_admins" {
+  name        = "SR-STG-ADMINS"
+  description = "STAGING Application Administrators for SimpleReport"
+}
+
+resource "okta_group" "simplereport_devtest_admins" {
+  name        = "SR-TEST-ADMINS"
+  description = "DEV/TEST Application Administrators for SimpleReport"
+}
+
```

Comment:
Is the expectation that non-administrative groups (e.g., facility staff) gets managed within Okta or Simple Report somehow?

Reply:
Within Okta, manually for now and eventually via API calls from SR.